packages/server/graphql/public/mutations/setMeetingMusic.ts

```diff
@@ -0,0 +1,25 @@
+import {SubscriptionChannel} from '../../../../client/types/constEnums'
+import publish from '../../../utils/publish'
+import {MutationResolvers} from '../resolverTypes'
+
+const setMeetingMusic: MutationResolvers['setMeetingMusic'] = async (
+  _source,
+  {meetingId, trackSrc, isPlaying, timestamp},
+  {dataLoader, socketId: mutatorId}
+) => {
+  const operationId = dataLoader.share()
+  const subOptions = {mutatorId, operationId}
+
+  const data = {
+    meetingId,
+    trackSrc: trackSrc || null,
+    isPlaying: !!isPlaying,
+    timestamp: timestamp || null
```

Comment:
+1 I guess we should store this on the meeting object so late comers can join in as well. Something like `{musicTrackSrc: string|null, musicStartedAt: timestamp|null}`